tx/textual/valuerenderer/valuerenderer.go

```diff
@@ -74,7 +77,12 @@ func (r *Textual) GetFieldValueRenderer(fd protoreflect.FieldDescriptor) (ValueR
 		fd.Kind() == protoreflect.Uint64Kind ||
 		fd.Kind() == protoreflect.Int32Kind ||
 		fd.Kind() == protoreflect.Int64Kind:
-		return NewIntValueRenderer(), nil
+		{
+			return NewIntValueRenderer(fd), nil
+		}
+
+	case fd.Kind() == protoreflect.StringKind:
```

Comment:
Why have this been re-added? This is a duplicate condition (see sonar result)

Reply:
Merge error probably, thanks for catching this.